app/src/lib/design-system/components/RQModal/RQModal.css

```diff
@@ -26,12 +26,17 @@
 }
 
 .rq-modal.ant-modal .rq-modal-content {
-  padding: 3.125rem;
-  border-top-left-radius: inherit;
-  border-top-right-radius: inherit;
+  padding: 3rem;
+  border-radius: inherit;
```

Comment:
Can you confirm that the extra modal does not change the modal elsewhere?

Reply:
yes